doc/connectivity/bluetooth/api/mesh/shell.rst

```diff
@@ -127,114 +130,122 @@ General configuration
-
-
-``mesh lpn <value: off, on>``
------------------------------
-
-	Enable or disable Low Power operation. Once enabled, the device will turn off its radio and start polling for friend nodes. The device will not be able to receive messages from the mesh network until the friendship has been established.
-
-	* ``value``: Sets whether Low Power operation is enabled.
-
-
-``mesh poll``
--------------
-
-	Perform a poll to the friend node, to receive any pending messages. Only available when LPN is enabled.
-
-
-``mesh ident``
---------------
-
-	Enable the Proxy Node Identity beacon, allowing Proxy devices to connect explicitly to this device. The beacon will run for 60 seconds before the node returns to normal Proxy beacons.
-
+	Reset the local mesh node to its initial unprovisioned state.
 
-``mesh dst [destination address]``
-----------------------------------
+``mesh target dst [destination address]``
+-----------------------------------------
 
 	Get or set the message destination address. The destination address determines where mesh packets are sent with the shell, but has no effect on modules outside the shell's control.
 
 	* ``destination address``: If present, sets the new 16-bit mesh destination address. If omitted, the current destination address is printed.
 
 
-``mesh netidx [NetIdx]``
-------------------------
+``mesh target net [NetIdx]``
+----------------------------
 
 	Get or set the message network index. The network index determines which network key is used to encrypt mesh packets that are sent with the shell, but has no effect on modules outside the shell's control. The network key must already be added to the device, either through provisioning or by a Configuration Client.
 
 	* ``NetIdx``: If present, sets the new network index. If omitted, the current network index is printed.
 
 
-``mesh appidx [AppIdx]``
-------------------------
+``mesh target app [AppIdx]``
+----------------------------
 
 	Get or set the message application index. The application index determines which application key is used to encrypt mesh packets that are sent with the shell, but has no effect on modules outside the shell's control. The application key must already be added to the device by a Configuration Client, and must be bound to the current network index.
 
 	* ``AppIdx``: If present, sets the new application index. If omitted, the current application index is printed.
 
+Low Power Node
+==============
 
-``mesh net-send <hex string>``
-------------------------------
+``mesh lpn set <value: off, on>``
+---------------------------------
 
-	Send a raw mesh message with the current destination address, network and application index. The message opcode must be encoded manually.
+	Enable or disable Low Power operation. Once enabled, the device will turn off its radio and start polling for friend nodes. The device will not be able to receive messages from the mesh network until the friendship has been established.
```

Comment:
I'm pretty sure device can receive messages from the mesh network. It depends on configuration options. Even more, lpn device has to be configured after provisioning.

Reply:
Sure 👍